### DISCUSSION TITLE
1.1.1

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#128] [Обновления]
**Известные ошибки:**
- Перевод Twitter не работает в cloudflare версии расширения (#119)

**Изменения:**
- Задержка до повторного запроса немного уменьшена (70 -> 60)
- Добавлено отображение времени до окончания перевода
- Сделана заготовка под Facebook. В этом обновление не получилось его добавить из-за того, что Facebook в последний момент что-то обновили и всё сломалось (появился CSP из-за которого расширение не может загрузить аудио с сайта яндекса)
- Изменен конфиг дебага
- Добавлены дебаг логи
- Немного почищен скрипт отслеживающий изменение в айдишнике для ВК
- Починена работа переводчика при смене страницы в твиттере и ВК (получилось костыльно, ну да ладно)
- Починено отображение кнопки в твиттере при смене страницы
- Исправлено добавление кнопки для мобильного твича (классы устарели)
- Добавлена поддержка твич клипов
- Немного переработан Readme-файл
- Добавлена возможность перевода с русского на английский на YouTube
- Добавлен автодетект языка видео для Youtube
- В протобаф ответа добавлено новое поле "Оставшееся время ожидания" (время до получения перевода)
- В протобаф запроса добавлено новое поле "Требуемый язык перевода" (это язык на который видео будет переведено)
- Добавлена поддержка Rutube
- Изменена стандартная команда сборки. Теперь, команда `build` собирает сразу две версии, команда `build:default` собирает обычную версию, а команда `build:cloudflare` собирает версию для cloudflare
- Возвращены локализированные описания расширений
- Webpack-userscript обновлен до 3.2.2. Теперь, cloudflare версия нормально собирается

## What's Changed
* vot-cloudflare.user.js : fix the auto-update link by @ACherepkov1989 in https://github.com/ilyhalight/voice-over-translation/pull/116

## New Contributors
* @ACherepkov1989 made their first contribution in https://github.com/ilyhalight/voice-over-translation/pull/116

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.1.0...1.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.1.1'>1.1.1</a>.</em>

**Comment by FiLinX** [comment]
хы.. даже вот интересно послушать, как оно будет на инглишь переводить моменты с обилием матюков))
киньте кто знает смачные видосики такие для теста)
